test/fixtures/dependency-check/tsx-pass/src/index.tsx

```diff
@@ -0,0 +1,8 @@
+// @ts-expect-error - not installed
+import React from 'react'
+
+// @ts-expect-error - not installed
+const Component: React.FC = () => (<div>Hello, world!</div>)
+const App: React.FC = () => (<Component />)
+
+export default App
```

Comment:
There's no TypeScript syntax in this file - I notice `.tsx` uses the same parser from `depcheck` as `.jsx`, please can you export an interface or something that would trip a js parser up?

Reply:
`const Component: React.FC` is a typescript type

Reply:
Ah, so it is.  My eyes are tired 👍